prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,42 @@
+const number1 = 10;
+const number2 = 4;
+let product = number1 * number2;
+console.log(product);
+console.log(typeof product);
+
+let charge = 5;
+let payment = 2;
+let amountRemaining = charge - payment;
+console.log(amountRemaining);
+console.log(typeof amountRemaining);
+
+const tests = 90;
+const assignments = 95;
+const final = 92;
+let grade = (tests + assignments + final) / 3;
+console.log(grade);
+console.log(typeof grade);
+
+// let arr = [tests, assignments, final];
+// console.log(arr.length);
+// console.log(arr.mean);
+
+const firstName = 'Joseph';
+const lastName = 'Park';
+const fullName = firstName + ' ' + lastName;
+console.log(fullName);
+console.log(typeof fullName);
+
+const pH = 7;
+const isAcidic = pH < 7;
+console.log(isAcidic);
+console.log(typeof isAcidic);
+
+const numberOfSoldiers = 500;
+const isSparta = numberOfSoldiers === 300;
+console.log(isSparta);
+console.log(typeof isSparta);
+
+let nickname = fullName + ' is the GOAT';
```

Comment:
This should be broken up into two steps. The first one should be assigning `fullName` to `nickName`. Then on another line, use the `+=` operator to add the string to `nickname`